### ISSUE TITLE
ALT text fallback color is inaccessible

### ISSUE THREAD

**Issue by BunnyNabbit** [BunnyNabbit/rayleigh-labels#31]
**Describe the bug**
Zhe ALT text color is black which is unreadable on zhe black background. 
**Expected behavior**
It should have displayed zhe ALT text in white or anozher distinct color. 
**Screenshots**
![Screenshot_20250407-002715-671.png](https://github.com/user-attachments/assets/789c72d8-cc92-4daa-a6a4-7f57b22edbc0)
